python/cross_service/aurora_item_tracker/README.md

```diff
@@ -0,0 +1,202 @@
+#  Tracking work items in a serverless Aurora database with the SDK for Python
+
+## Overview
+
+This example shows you how to use the AWS SDK for Python (Boto3) to create a REST 
+service that lets you do the following:
+
+* Read, write, and update work items that are stored in a serverless Amazon Aurora
+  database.
```

Comment:
Question: Here, and throughout the other files in this PR, there are mentions of a "serverless Amazon Aurora database." Are these references to an Amazon Aurora Serverless database?

Reply:
Yes. Fixed this one and searched for (and fixed) the rest.